*** ./smali/android/app/ActivityThread.smali	2017-10-30 11:16:29.620262969 +0100
--- /home/mcfy/bin/device/temp/new_smali/framework.jar.out/./smali/android/app/ActivityThread.smali	2017-10-30 11:07:16.263690432 +0100
***************
*** 6345,6350 ****
--- 6345,6356 ----
      invoke-virtual {v2}, Ldalvik/system/VMRuntime;->clearGrowthLimit()V
  
      :goto_a
+     move-object/from16 v0, p1
+ 
+     iget-object v2, v0, Landroid/app/ActivityThread$AppBindData;->appInfo:Landroid/content/pm/ApplicationInfo;
+ 
+     invoke-static {v2}, Lcom/miui/whetstone/app/WhetstoneAppManager;->trimHeapSizeIfNeeded(Landroid/content/pm/ApplicationInfo;)V
+ 
      invoke-static {}, Landroid/os/StrictMode;->allowThreadDiskWrites()Landroid/os/StrictMode$ThreadPolicy;
  
      move-result-object v26
